client/src/components/musicPlayer.jsx

```diff
@@ -1,24 +1,178 @@
+const TitleArtistContainer = styled.div`
+  padding-left: 10px;
+  display: flex;
+  flex-direction: column; 
+  width: 400px;
+  font-weight: 100;
+`;
+
+const PlayButtonContainer = styled.div`
+  justify-content: flex-start;
+  border-radius: 50%;
+  border-color: #FF5836;
+  background: #FF5836;
+  width: 60px;
+  height: 60px;
+  cursor: pointer;
+  &:hover {
+    background: #FF4137;
+  }
+`;
+const PlayButton = styled.img`
+  margin-left: 7px;
+  margin-top: 4px;
+  width: 50px;
+  height: 50px;
+`;
+
+const ArtistContainer = styled.div`
+  
+`;
+
+const AlbumContainer = styled.div`
+  
+`;
+
+const AlbumArtContainer = styled.div`
+  width: 340px;
+  height: 340px;
+`;
+
+const GenreCreatedContainer = styled.div`
+  margin-left: auto;
+  margin-right: 20px;
+  flex-direction: column;
+  font-size: 16px;
+`;
+
+const CreatedAt = styled.div`
+  margin-top: 16px;
+  margin-bottom: 16px;
+`;
+
+const GenreContainer = styled.div`
+  flex-direction: row;
+  text-align: right;
+`;
+
+const Artist = styled.span`
+  background-color: black;
+  font-size: 16px;
+  color: #ccc;
+  padding-top: 2px;
+  padding-right: 7px;
+  padding-bottom: 3px;
+  padding-left: 7px;
+  cursor: pointer;
+  ${ArtistContainer}:hover &{
+    color: white;
+  }
+`;
+
+const Album = styled.span`
+  background-color: black;
+  font-size: 16px;
+  color: #ccc;
+  line-height: 1;
+  padding-top: 2px;
+  padding-right: 7px;
+  padding-bottom: 3px;
+  padding-left: 7px;
+  cursor: pointer;
+  ${AlbumContainer}:hover &{
+    color: white;
+  }
+`;
+
+const Genre = styled.span`
+  background: #999;
+  color: #fff;
+  cursor: pointer;
+  border-radius:20px;
+  border-top-color: rgb(153, 153, 153);
+  border-top-style: solid;
+  border-top-width: 1px;
+  border-right-color: rgb(153, 153, 153);
+  border-right-style: solid;
+  border-right-width: 1px;
+  border-bottom-color: rgb(153, 153, 153);
+  border-bottom-style: solid;
+  border-bottom-width: 1px;
+  border-left-color: rgb(153, 153, 153);
+  border-left-style: solid;
+  border-left-width: 0;
+  border-image-source: initial;
+  border-image-slice: initial;
+  border-image-width: initial;
+  border-image-outset: initial;
+  border-image-repeat: initial;
+  padding-top: 2px;
+  padding-right: 7px;
+  padding-bottom: 2px;
+  padding-left: 7px;
+  ${GenreContainer}:hover &{
+    background: #666;
+    border-color: #666;
+  }
+`;
+
+const AlbumArt = styled.img`
+  width: 340px;
+  height: 340px;
+`;
+
+
+const Title = styled.span`
+  background-color: black;
+  font-size: 24px;
+  color: white;
+  line-height: 2.1;
+  padding-top: 4px;
+  padding-right: 7px;
+  padding-bottom: 4px;
+  padding-left: 7px;
+
+`;
 
 class MusicPlayer extends React.Component {
   constructor (props) {
     super(props);
     this.state = {
       currentSongData: {
-        "id": 1,
-        "title": "reprehenderit marfa quinoa bag",
-        "artist": "Rick Astley",
-        "genre": "Jazz",
-       "album": "Whenever You Need Sombeody"
+        id: 1,
```

Comment:
instead of hardcoding it, might it be better to do an AJAX request upon component mount and just display the first result?

Reply:
They were initially erroring without those, but I have now since (at a later commit) done away with this.